modules/rosa-sts-account-roles-terraform.adoc

```diff
@@ -0,0 +1,246 @@
+  }
+
+  variable "account_role_prefix" {
+    type    = string
+    default = ""
+    description = "Your account roles are prepended with whatever value you enter here. The default value in the ROSA CLI is 'ManagedOpenshift-' before all of your account roles."
+  }
+
+  variable "tags" { <1>
+    type        = map
+    default     = null
+    description = "(Optional) List of AWS resource tags to apply."
+  }
+EOF
+----
++
+--
+<1> The `tags` parameter uses a map of strings variable. The format that it takes looks like the following example:
++
+[source,terraform]
+----
+variable "tags" {
+  type    = "map"
+  default = {
+    "us-east-1" = "image-1234"
+    "us-west-2" = "image-4567"
+  }
+}
+----
+--
+. In the directory where you saved these Terraform files, run the following command to set up Terraform to create these resources:
++
+[source,terminal]
+----
+$ terraform init
+----
+. Optional: Run the following command to confirm that the Terraform code you copied is correct:
++
+[source,terminal]
+----
+$ terraform validate
+----
++
+.Sample output
++
+[source,terminal]
+----
+Success! The configuration is valid.
+----
+. Optional: Test your Terraform template and create a reusable Terraform plan file by running the following command:
++
+[source,terminal]
+----
+$ terraform plan -out account-roles.tfplan -var token=<OCM_token>
+----
+. Run the following command to build your account-wide IAM roles with Terraform:
++
+[source,terminal]
+----
+$ terraform apply "account-roles.tfplan"
+----
++
+[NOTE]
+====
+If you used the `terraform plan` command first, you can provide your created `account-roles.tf` file here. Otherwise, Terraform temporarily creates this plan before it applies your desired outcome.
+====
+
+.Verification
+* Run the following command to verify that your account-roles have been created:
++
+[source,terminal]
+----
+$ rosa list account-roles
+----
++
+.Sample output
+
+[source,terminal]
+----
+I: Fetching account roles
+ROLE NAME                            ROLE TYPE      ROLE ARN                                                            OPENSHIFT VERSION  AWS Managed
+account-role-6kn4-ControlPlane-Role  Control plane  arn:aws:iam::269733383066:role/account-role-6kn4-ControlPlane-Role  4.13               No
+account-role-6kn4-Installer-Role     Installer      arn:aws:iam::269733383066:role/account-role-6kn4-Installer-Role     4.13               No
+account-role-6kn4-Support-Role       Support        arn:aws:iam::269733383066:role/account-role-6kn4-Support-Role       4.13               No
+account-role-6kn4-Worker-Role        Worker         arn:aws:iam::269733383066:role/account-role-6kn4-Worker-Role        4.13               No
+----
+
+.Clean up
+
+When you are finished using the resources that you created using Terraform, you should purge these resources with the following command:
+[source,terminal]
+----
+$ terraform destroy
```

Comment:
For the record,  this also requires specifying the token variable, so recommend changing to:

```
terraform destroy -var token=$RHCS_TOKEN
```